.azurepipelines/Platform-Build-GCC5.yml

```
@@ -176,3 +177,14 @@ jobs:
           **/QEMU_EFI.fd
           **/SECURE_FLASH0.fd
         BuildArtifactsOther: "**/unit_test_results/*"
+
+      QemuSbsa_CODE_COVERAGE:
+        BuildPackage: QemuSbsaPkg
+        BuildFile: "Platforms/QemuSbsaPkg/Test/PlatformTest.py"
+        BuildArch: "AARCH64"
+        BuildFlags: "CODE_COVERAGE=TRUE CC_FULL=TRUE CC_FLATTEN=TRUE"
+        BuildTarget: "NOOPT"
+        BuildExtraTag: "UNIT_TESTS"
+        BuildExtraStep:
+          - script: echo no steps provided

```

I think this is a bit more clear:

```suggestion
          - script: echo no extra steps provided

```

---

Updated for clarity. Thanks!